src/app/components/layouts/expandable-text/expandable-text.component.ts

```diff
@@ -0,0 +1,84 @@
+import {
+    AfterViewInit,
+    Component,
+    EventEmitter,
+    Input,
+    Output,
+} from '@angular/core';
+import { TranslationService } from 'angular-l10n';
+import { isNullOrUndefined } from 'util';
+
+
+let nextId:number = 0;
+
+@Component({
+    selector: 'tc-expandable-text',
+    styles:   [require('./expandable-text.component.scss')],
+    template: require('./expandable-text.component.html')
+})
+export class ExpandableTextComponent implements AfterViewInit
+{
+    @Input()
+    public collapsed:boolean = true;
+
+    @Input()
+    public text:string;
+
+    @Input()
+    public visibleLines:number = 1;
```

Comment:
default vllt undefined, d.h. es werden alle Zeilen angezeigt?
außerdem würde ich sagen reicht `lines` als name
```suggestion
    public lines:number = 1;
```

Reply:
lines, ja. undefined nicht, sonst müsste expanded = true sein

Reply:
ausserdem wäre bei expanded = true sowieso egal was bei lines steht da anderer html code angezeigt wird